requirements-build.txt

```diff
@@ -1,4 +1,4 @@
 # Requirements we need to run our build jobs for the installers.
 # We create the separation for cases where we're doing installation
 # from a local dependency directory instead of requirements.txt.
-PyInstaller==5.3
+PyInstaller==5.7.0
```

Comment:
Let's decouple upgrading PyInstaller from this change. Mainly we should make this change scoped to purely adding source support for Python 3.11 as the PyInstaller update will need additional vetting and potentially additional code changes.

Reply:
Hmm it looks like PyInstaller has a ceiling version pin on Python for the version of PyInstaller that we use and it excludes Python 3.11. We will probably need to upgrade PyInstaller first before we can get CI passing here.